.circleci/config.yml

```diff
@@ -630,6 +630,10 @@ jobs:
     steps:
       - checkout
       - *build_ubuntu_package
+      - persist_to_workspace:
```

Comment:
What is the intended result? I am not sure whether it is this or `- *store_package_artifacts`.